### PR TITLE
feat(content): add PR checklist to the GitHub PR section

### PR DESCRIPTION
**Pull request by kierisi**
added a PR checklist, as discussed with @lwasser. this list can be expanded upon, but represents a good initial set of checks for individuals to use when contributing to a pyopensci repository.

### CODE REVIEW

**Comment by kierisi** [comment]
pre-commit.ci autofix

File: community/github/pull-requests.md [hunk 1]
```diff
@@ -29,6 +29,23 @@ You might be surprised that you notice things in the pull request that you
 didn't notice when working on the content locally.
 :::
 
+### Pull Request Checklist
+
+The following checklist might be helpful in reviewing your own PR. This checklist is what we use internally before submitting a PR, in order to help ensure things go as smoothly as possible:
```
Comment (lwasser) [inline community/github/pull-requests.md:34]:
```suggestion
Please follow this pull request review checklist when submitting a PR to the pyOpenSci GitHub repository. This checklist will both make the pull request review process easier both for you as the PR author and the repository maintainers. It also will make the pull request merge process more smooth and efficient. 
```

File: organization/canva.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# pyOpenSci uses Canva as a shared graphic platform
```
Comment (lwasser) [inline organization/canva.md:1]: Is the cCanva content in this PR intentional? there is a merge conflict with it (we merged #74 ) in this PR.

**Review by lwasser — CHANGES_REQUESTED** [review]
@kierisi i haven't left a full set of comments. there is a merge conflict on this PR and also I don't know if you meant to add the canva page to this PR. can you please rebase this branch off of main and determine whether the canva page is intentional or not? I will then continue to review the content here. Feel free to address the few comments that i've already suggested and then ping me here to review again.

**Comment by kierisi** [comment]
closing in favor of #91

**Pull request closed without merging**